### PR TITLE
CorrectForm for apostrophe

### LINKED ISSUE #268: CorrectForm
To make this consistent, word 9 should have `CorrectForm='` in its MISC field.
```
# sent_id = reviews-041925-0002
# text = I will be calling tropics for my companies next event!
8-9	companies	_	_	_	_	_	_	_	_
8	companie	company	NOUN	NN	Number=Sing|Typo=Yes	11	nmod:poss	11:nmod:poss	CorrectForm=companies
9	s	's	PART	POS	Typo=Yes	8	case	8:case	_
```

Word 19 and 28 should have `Typo=Yes` and `CorrectForm=n't`, shouldn't they?  Word 2 can have `Abbr` or `Typo` as well.
```
# sent_id = answers-20111108102621AA3hPqj_ans-0005
# text = iv owned beardies for ovr 3 yrs and i handle mine all the time.. they dont go hard like needle hard but they arent pillow soft.
1-2	iv	_	_	_	_	_	_	_	_
1	i	I	PRON	PRP	Case=Nom|Number=Sing|Person=1|PronType=Prs	3	nsubj	3:nsubj	_
2	v	have	AUX	VBP	Mood=Ind|Number=Sing|Person=1|Tense=Pres|VerbForm=Fin	3	aux	3:aux	_
...
18-19	dont	_	_	_	_	_	_	_	_
18	do	do	AUX	VBP	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	20	aux	20:aux	_
19	nt	not	PART	RB	_	20	advmod	20:advmod	_
...
27-28	arent	_	_	_	_	_	_	_	_
27	are	be	AUX	VBP	Abbr=Yes|Mood=Ind|Number=Plur|Person=3|Tense=Pres|Typo=Yes|VerbForm=Fin	30	cop	30:cop	CorrectForm=are
28	nt	not	PART	RB	_	30	advmod	30:advmod	_
```
Similar cases (arent, wasnt, theyre):
```
# sent_id = answers-20111108092643AAXe4lD_ans-0063
# sent_id = reviews-100592-0003
# sent_id = reviews-220214-0001
# sent_id = answers-20111107155845AAE3kCA_ans-0002
```

In this case, `ur`'s expected surface strings are given, but how can we know the original sentence should be `you are` rather than `youare`?  Because there is no way to know the spaces in MWT.  Can the token 17-18 have `CorrectForm=you are`?
```
# sent_id = answers-20111108102621AA3hPqj_ans-0007
# text = they puff out to look bigger and badder and they can scratch you a lil if ur holding them to tight :)
17-18	ur	_	_	_	_	_	_	_	_
17	u	you	PRON	PRP	Typo=Yes	19	nsubj	19:nsubj	CorrectForm=you
18	r	be	AUX	VBP	Mood=Ind|Number=Sing|Person=3|Tense=Pres|Typo=Yes|VerbForm=Fin	19	cop	19:cop	CorrectForm=are
```

A similar case (their->they are):
```
# sent_id = answers-20111108093211AA8bYFE_ans-0002
# text = I'm getting a dwarf Hamster tomorrow And i was just talking to my Mom and she said that we have to take out the wheel at night because it would disturb her sleep But i told her no because i said their energetic at night at my room is right next door to hers But i heard their are wheels that dont make noise at all what would be the price of it?
43-44	their	_	_	_	_	_	_	_	_
43	thei	they	PRON	PRP	Typo=Yes	45	nsubj	45:nsubj	CorrectForm=they
44	r	be	AUX	VBP	Mood=Ind|Number=Sing|Person=3|Tense=Pres|Typo=Yes|VerbForm=Fin	45	cop	45:cop	CorrectForm=are
```

### PR DESCRIPTION
**Pull request by kanayamah**
Add `CorrectForm='s` for missing apostrophe cases.

Closes #268.

### CODE REVIEW

File: en_ewt-ud-test.conllu [hunk 1]
```diff
@@ -9369,20 +9369,20 @@
...
+3	:	:	PUNCT	:	_	5	punct	5:punct	CheckAttachment=4
+4	John	john	X	GW	_	5	compound	5:compound	CheckAttachment=1|CheckReln=obl
+5	Salinardo@ENRON	salinardo@enron	X	ADD	_	1	obl	1:obl:by	CheckAttachment=4|CheckReln=goeswith
 
 # sent_id = email-enronsent28_01-0004
 # text = it is to late for me to add changes.
-1	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	4	nsubj	4:nsubj	_|CheckReln=expl
+1	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	4	nsubj	4:nsubj	CheckReln=expl
 2	is	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	cop	4:cop	_
 3	to	to	ADV	RB	_	4	advmod	4:advmod	_
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:9380]: typo: too

File: en_ewt-ud-test.conllu [hunk 2]
```diff
@@ -9369,20 +9369,20 @@
...
-3	:	:	PUNCT	:	_	5	punct	5:punct	_|CheckAttachment=4
-4	John	john	X	GW	_	5	compound	5:compound	_|CheckAttachment=1|CheckReln=obl
-5	Salinardo@ENRON	salinardo@enron	X	ADD	_	1	obl	1:obl:by	_|CheckAttachment=4|CheckReln=goeswith
+3	:	:	PUNCT	:	_	5	punct	5:punct	CheckAttachment=4
+4	John	john	X	GW	_	5	compound	5:compound	CheckAttachment=1|CheckReln=obl
+5	Salinardo@ENRON	salinardo@enron	X	ADD	_	1	obl	1:obl:by	CheckAttachment=4|CheckReln=goeswith
 
 # sent_id = email-enronsent28_01-0004
 # text = it is to late for me to add changes.
-1	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	4	nsubj	4:nsubj	_|CheckReln=expl
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:9378]: I think this should be `expl` and "add" should attach as `advcl`. See "it is too early to say whether Bush’s tough message was instrumental"

File: en_ewt-ud-test.conllu [hunk 3]
```diff
@@ -9403,8 +9403,8 @@
 # text = Thank you for you patience.
 1	Thank	thank	VERB	VBP	Mood=Ind|Number=Sing|Person=1|Tense=Pres|VerbForm=Fin	0	root	0:root	_
 2	you	you	PRON	PRP	Case=Acc|Person=2|PronType=Prs	1	obj	1:obj	_
-3	for	for	ADP	IN	_	5	case	5:case	_|CheckAttachment=4
-4	you	you	PRON	PRP	Case=Nom|Person=2|PronType=Prs	5	nmod:poss	5:nmod:poss	_|CheckAttachment=1|CheckReln=obl
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:9407]: typo: your

File: en_ewt-ud-test.conllu [hunk 4]
```diff
@@ -9698,8 +9698,8 @@
 # sent_id = email-enronsent28_01-0021
 # newpar id = email-enronsent28_01-p0004
 # text = DEBRA PERLINGIERE
-1	DEBRA	DEBRA	PROPN	NNP	Number=Sing	2	compound	2:compound	_|CheckAttachment=0|CheckReln=root
-2	PERLINGIERE	PERLINGIERE	PROPN	NNP	Number=Sing	0	root	0:root	_|CheckAttachment=1|CheckReln=flat
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:9702]: should be `flat`

File: en_ewt-ud-test.conllu [hunk 5]
```diff
@@ -20464,7 +20464,7 @@
 4-6	Mc.Donalds	_	_	_	_	_	_	_	SpaceAfter=No
 4	Mc.	Mc.	PROPN	NNP	Number=Sing	5	compound	5:compound	_
 5	Donald	Donald	PROPN	NNP	Number=Sing	3	nsubj	3:nsubj	_
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:20466]: should be `goeswith`

File: en_ewt-ud-test.conllu [hunk 6]
```diff
@@ -23268,7 +23268,7 @@
 29	mc	mc	PROPN	NNP	Number=Sing	30	compound	30:compound	_
 30-31	donalds	_	_	_	_	_	_	_	_
 30	donald	donald	PROPN	NNP	Number=Sing	27	obl	27:obl:s	_
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:23270]: should be `goeswith`

File: en_ewt-ud-test.conllu [hunk 7]
```diff
@@ -23448,7 +23448,7 @@
 # text = Fudruckers.
 1-2	Fudruckers	_	_	_	_	_	_	_	SpaceAfter=No
 1	Fudrucker	Fudrucker	PROPN	NNP	Number=Sing	0	root	0:root	_
-2	s	's	PART	POS	Typo=Yes	1	case	1:case	_
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:23451]: [Fuddruckers](https://en.wikipedia.org/wiki/Fuddruckers) is actually not possessive, but is spelled with two d's.

**Comment by nschneid** [comment]
Thanks, this is great. Would you mind first doing a PR just to remove the `_|` prefixes in the MISC column (not sure how those got there)? That will make it easier to review all the possessive corrections.

**Comment by kanayamah** [comment]
@nschneid Hi, thank you for comment.  Actually I was not aware of redundant `_|` in MISC field -- it is automatically fixed when I re-serialized using script.  I'll raise another issue to fix it.

**Comment by kanayamah** [comment]
@nschneid Done: please see #273 #274.  After merging it let's come back here.

File: en_ewt-ud-dev.conllu [hunk 8]
```diff
@@ -22227,7 +22227,7 @@
 14	almost	almost	ADV	RB	_	17	advmod	17:advmod	_
 15-16	valentines	_	_	_	_	_	_	_	_
 15	valentine	valentine	PROPN	NNP	Number=Sing	17	nmod:poss	17:nmod:poss	_
-16	s	's	PART	POS	Typo=Yes	15	case	15:case	_
+16	s	's	PART	POS	Typo=Yes	15	case	15:case	CorrectForm='s
 17	day	day	NOUN	NN	Number=Sing	4	conj	4:conj:but	_
 18	afnd	and	CCONJ	CC	_	21	cc	21:cc	_
```
Comment (nschneid) [inline en_ewt-ud-dev.conllu:22232]: Should have `Typo=Yes`.

File: en_ewt-ud-dev.conllu [hunk 9]
```diff
@@ -22268,7 +22268,7 @@
 54	a	a	DET	DT	Definite=Ind|PronType=Art	55	det	55:det	_
 55-56	guys	_	_	_	_	_	_	_	_
 55	guy	guy	NOUN	NN	Number=Sing	57	nmod:poss	57:nmod:poss	_
-56	s	's	PART	POS	Typo=Yes	55	case	55:case	_
+56	s	's	PART	POS	Typo=Yes	55	case	55:case	CorrectForm='s
 57	help	help	NOUN	NN	Number=Sing	53	obj	53:obj	_
 58	what	what	PRON	WP	PronType=Int	61	obj	61:obj	_
 59	shoul	should	AUX	MD	VerbForm=Fin	61	aux	61:aux	_
```
Comment (nschneid) [inline en_ewt-ud-dev.conllu:22274]: Should have `Typo=Yes`

File: en_ewt-ud-test.conllu [hunk 10]
```diff
@@ -15757,7 +15757,7 @@
 9	,	,	PUNCT	,	_	12	punct	12:punct	_
 10-11	Dinners	_	_	_	_	_	_	_	_
 10	Dinner	Dinner	PROPN	NNP	Number=Sing	12	nmod:poss	12:nmod:poss	_
-11	s	's	PART	POS	Typo=Yes	10	case	10:case	_
+11	s	's	PART	POS	Typo=Yes	10	case	10:case	CorrectForm='s
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:15760]: Typo: should be [Diners Club](https://en.wikipedia.org/wiki/Diners_Club_International), no possessive

File: en_ewt-ud-train.conllu [hunk 11]
```diff
@@ -152575,7 +152575,7 @@
 11	ur	ur	PRON	PRP$	_	12	nmod:poss	12:nmod:poss	_
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:152575]: Abbr=Yes

**Comment by nschneid** [comment]
Thanks @kanayamah! I'll make a separate PR for the miscellaneous issues still open above.

**Pull request merged**